src/kirby/components/empty-state/empty-state.component.tns.html

```diff
@@ -0,0 +1,10 @@
+<FlexboxLayout flexDirection="column" justifyContent="space-around">
+  <StackLayout class="box" alignSelf="center">
+    <kirby-icon *ngIf="iconName" [name]="iconName" size="lg" outline="true" class="icon"></kirby-icon>
+    <Label *ngIf="title" class="kirby-text-normal-bold title" [text]="title"></Label>
+    <Label *ngIf="text" class="kirby-text-normal message" [text]="text" textWrap="true"></Label>
+    <StackLayout class="button-wrapper">
```

Comment:
Do you need to wrap the content?
```suggestion
     <ng-content></ng-content>
```

Reply:
This is needed for us to be able to apply a margin-top to the content.